code/modules/projectiles/projectile/beams.dm

```diff
@@ -55,20 +61,32 @@
 		impact_effect_type = /obj/effect/temp_visual/impact_effect/red_laser/wall
 
 /obj/projectile/beam/weak
+	tracer_type = /obj/effect/projectile/tracer/laser
+	muzzle_type = /obj/effect/projectile/muzzle/laser
+	impact_type = /obj/effect/projectile/impact/laser
```

Comment:
am i going crazy aren't these 3 already the default values on projectile/beam

Reply:
well, I left things like pulse be for now, but I can make it all beam

Reply:
No i'm saying on the base `obj/projectile/beam` class the `tracer_type` variable is already set to `/obj/effect/projectile/tracer/laser`, (same with muzzle_type and impact_type). Why are you setting the variables again here, in `obj/projectile/beam/weak`?

Reply:
the reason is that I didn't set it in the main obj/projectile/beam beforehand. also its just in case, because last time there was no traces/beams visible, but it worked